tox.ini

```diff
@@ -33,21 +33,28 @@ basepython = python3.7
 skip_install = true
 deps =
     bandit==1.7.0
-    black==21.5b1
+    black==21.6b0
     flake8==3.9.2
     flake8-bugbear==21.4.3
     flake8-comprehensions==3.5.0
     flake8-docstrings==1.6.0
-    isort==5.8.0
-    mypy==0.812
+    isort==5.9.1
+    mypy==0.910
     pandas-stubs==1.1.0.6
     sqlalchemy-stubs==0.4
+    types-PyYAML==5.4.3
+    types-requests==2.25.0
+    types-setuptools==57.0.0
+    typing-extensions==3.10.0.0
 commands =
     flake8 {posargs:{[tox]sources}}
     isort --check {posargs:{[tox]sources}}
     black --check {posargs:{[tox]sources}}
-    bandit -qr --exclude "benchmarks/,tests/" {posargs:{[tox]sources}}
-    mypy --exclude "benchmarks/conftest.py" {posargs:{[tox]sources}}
+    bandit -qr --exclude "benchmarks/,tests/,data_and_models/pipelines/ner/transformers_vs_spacy/transformers/" \
+    {posargs:{[tox]sources}}
+    mypy \
+    --exclude "benchmarks/conftest.py|data_and_models/pipelines/ner/transformers_vs_spacy/transformers/|data_and_models/pipelines/sentence_embedding/scripts/" \
```

Comment:
I think the exclude configuration can also be put into the `.mypy.ini` file. Is there anything that speaks against it?

I think the advantage would be that the setting would also apply when you run `mypy` from the command line.

Reply:
Done in 99b63f4.